File: src/main/java/com/peerly/peerly_server/models/dto/CreateSessionRequest.java
package com.peerly.peerly_server.models.dto;

import com.fasterxml.jackson.annotation.JsonFormat;
import java.time.LocalDateTime;

public record CreateSessionRequest(
    String tutorId,
    String subjectId,
    String title,
    String description,
    @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss") LocalDateTime startsAt,
    @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss") LocalDateTime endsAt,
    Integer maxParticipants,
    Integer priceTotalCents,
    String studentId
) {}

File: app/src/main/java/com/example/Peerly/data/model/CreateSessionRequest.kt
package com.example.Peerly.data.model

data class CreateSessionRequest(
    val tutorId: String,
    val subjectId: String? = null,
    val title: String,
    val description: String? = null,
    val startsAt: String,
    val endsAt: String,
    val maxParticipants: Int? = 1,
    val priceTotalCents: Int? = null,
    val studentId: String? = null
)
